### PR TITLE
dnf, lsmod, lsusb: add Spanish translation

### PR DESCRIPTION
**Pull request by 1-byte-man**
Translated 3 tldr pages

### CODE REVIEW

File: pages.es/linux/dnf.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dnf
+
+> Administrador de paquetes para RHEL, Centos y Fedora (Reemplaza a yum).
```
Comment (navarroaxel) [inline pages.es/linux/dnf.md:3]:
```suggestion
> Administrador de paquetes para RHEL, CentOS y Fedora (Reemplaza a yum).
```

File: pages.es/linux/dnf.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> Más información en https://dnf.readthedocs.io
```
Comment (navarroaxel) [inline pages.es/linux/dnf.md:4]:
```suggestion
> Más información: <https://dnf.readthedocs.io>.
```

File: pages.es/linux/dnf.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Actualiza todos los paquetes a la última versión disponible:
+
+`sudo dnf update`
+
+- Busca un paquete según palabras clave:
```
Comment (navarroaxel) [inline pages.es/linux/dnf.md:10]:
```suggestion
- Busca un paquete usando palabras clave:
```

File: pages.es/linux/dnf.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`dnf search {{palabra clave}}`
```
Comment (navarroaxel) [inline pages.es/linux/dnf.md:12]: We use `snake_case` for variable names:
```suggestion
`dnf search {{palabra_clave}}`
```

File: pages.es/linux/lsmod.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# lsmod
+
+> Muestra el estado de los módulos cargados en el kernel de linux. Vease tambien `modprobe` el cual carga módulos de kernel
```
Comment (navarroaxel) [inline pages.es/linux/lsmod.md:3]: I think that `,` is required for the correct grammar:
```suggestion
> Muestra el estado de los módulos cargados en el kernel de linux.
> Vease tambien `modprobe`, el cual carga módulos de kernel.
```

File: pages.es/linux/lsmod.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+- liste todos los módulos con:
```
Comment (navarroaxel) [inline pages.es/linux/lsmod.md:5]:
```suggestion
- Lista todos los módulos de kernel cargados: 
```

File: pages.es/linux/lsusb.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# lsusb
+
+> Muestra información sobre puertos y dispositivos USB
```
Comment (navarroaxel) [inline pages.es/linux/lsusb.md:3]:
```suggestion
> Muestra información sobre puertos y dispositivos USB.
```

File: pages.es/linux/lsusb.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+- Lista todos los dispositivos USB disponibles:
+
+`lsusb`
+
+- Lista la jerarquía de dispositivos USB en forma de arbol
```
Comment (navarroaxel) [inline pages.es/linux/lsusb.md:9]:
```suggestion
- Lista la jerarquía de dispositivos USB en forma de árbol:
```

File: pages.es/linux/lsusb.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+`lsusb -t`
+
+- Lista los dispositivos USB de forma verbosa:
+
+`lsusb --verbose`
+
+- Lista información destallada acerca de un dispositivo USB en especifico:
```
Comment (navarroaxel) [inline pages.es/linux/lsusb.md:17]:
```suggestion
- Lista información detallada acerca de un dispositivo USB determinado:
```

File: pages.es/linux/lsusb.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+`lsusb -D {{dispositivo}}`
+
+- Lista solo dispositivos con un ID de ensamblador y producto en especifico:
```
Comment (navarroaxel) [inline pages.es/linux/lsusb.md:21]:
```suggestion
- Lista solo dispositivos con un ID de ensamblador y producto determinado:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @1-byte-man, welcome and thanks for your submission. I left some comments.

**Review by navarroaxel — APPROVED** [review]
Thanks @1-byte-man.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**